indico/modules/events/registration/templates/display/participant_list.html

```diff
@@ -52,105 +52,23 @@
-                    {% else %}
-                        <div style="height: 64px;"></div>
-                    {% endif %}
-                </td>
-            {% else %}
-                <td class="i-table"
-                    {% if column.sort_key %} data-text="{{ column.sort_key }}"{% endif %}>
-                    {{ column.text }}
-                </td>
-            {% endif %}
-        {% endfor %}
-{% endmacro %}
-
-{% macro participant_table(table) %}
-    <section>
-        {% set hidden_participants = table.num_participants - table.rows|length %}
-        {% if table.title %}
-            <div class="header">
-                <div class="header-row">
-                    <h3>{{ table.title }}</h3>
-                </div>
-            </div>
-        {% endif %}
-        {% if table.rows %}
-            <table class="i-table tablesorter">
-                {{ table_header(table.headers, table.show_checkin) }}
-                <tbody>
-                    {% for row in table.rows %}
-                        {{ table_row(row, table.show_checkin) }}
-                    {% endfor %}
-                    {% if hidden_participants %}
-                        <tr class="i-table">
-                            <td class="i-table empty" colspan="{{ table.headers|length+1 if table.show_checkin else table.headers|length }}">
-                                {% trans num=hidden_participants %}{{ num }} more participant{% pluralize %}{{ num }} more participants{% endtrans %}
-                            </td>
-                        </tr>
-                    {% endif %}
-                </tbody>
-            </table>
-        {% elif table.num_participants %}
-            {% call message_box('info', fixed_width=true) %}
-                {% trans num=hidden_participants %}There is {{ num }} hidden participant{% pluralize %}There are {{ num }} hidden participants{% endtrans %}
-            {% endcall %}
-        {% else %}
-            {% call message_box('info', fixed_width=true) %}
-                {% trans %}There are no registrations yet.{% endtrans %}
-            {% endcall %}
-        {% endif %}
-    </section>
-{% endmacro %}
-
 {% block content %}
-    {% if not published or not tables %}
-        {% call message_box('info', fixed_width=true) %}
-            {% trans %}There are no published registrations.{% endtrans %}
-        {% endcall %}
-    {% elif not num_participants %}
-        {% call message_box('info', fixed_width=true) %}
-            {% trans %}There are no registrations yet.{% endtrans %}
-        {% endcall %}
-    {% else %}
-        <div class="registrations list">
-            {% for table in tables %}
-                {{ participant_table(table) }}
-            {% endfor %}
-        </div>
-    {% endif %}
+    <div
+        id="participant-list-accordion"
+        data-tables="{{ tables | tojson | forceescape }}"
+        {% if published %}
+            data-published="true"
```

Comment:
for bool data attributes it's more common to just set `data-published` w/o a value, and check for `!== undefined` when reading it.

Reply:
In fact, `published` after reading it from the `dataset` will be either `undefined` or `'true'`, but never a boolean. So if someone were to put e.g. `data-published="false"` in here, it would *still* be truthy --> just avoid boolean data attributes with a value